opencti-platform/opencti-front/src/private/components/settings/status_templates/StatusTemplates.tsx

```diff
@@ -1,118 +1,173 @@
-import StatusTemplateLineDummy from './StatusTemplateLineDummy';
 import { StatusTemplatesLinesPaginationQuery, StatusTemplatesLinesPaginationQuery$variables } from './__generated__/StatusTemplatesLinesPaginationQuery.graphql';
 import LabelsVocabulariesMenu from '../LabelsVocabulariesMenu';
 import { useFormatter } from '../../../../components/i18n';
 import Breadcrumbs from '../../../../components/Breadcrumbs';
 import useConnectedDocumentModifier from '../../../../utils/hooks/useConnectedDocumentModifier';
+import { emptyFilterGroup, useBuildEntityTypeBasedFilterContext } from '../../../../utils/filters/filtersUtils';
+import { DataTableProps } from '../../../../components/dataGrid/dataTableTypes';
+import { UsePreloadedPaginationFragment } from '../../../../utils/hooks/usePreloadedPaginationFragment';
+import DataTable from '../../../../components/dataGrid/DataTable';
 
-// Deprecated - https://mui.com/system/styles/basics/
-// Do not use it for new code.
-const useStyles = makeStyles(() => ({
-  container: {
-    margin: 0,
-    padding: '0 200px 50px 0',
-  },
-}));
+export const statusTemplatesQuery = graphql`
+  query StatusTemplatesLinesPaginationQuery(
+    $search: String
+    $count: Int!
+    $cursor: ID
+    $orderBy: StatusTemplateOrdering
+    $orderMode: OrderingMode
+  ) {
+    ...StatusTemplatesLines_data
+    @arguments(
+      search: $search
+      count: $count
+      cursor: $cursor
+      orderBy: $orderBy
+      orderMode: $orderMode
+    )
+  }
+`;
+
+export const statusTemplatesFragment = graphql`
+  fragment StatusTemplatesLines_data on Query
+  @argumentDefinitions(
+    search: { type: "String" }
+    count: { type: "Int", defaultValue: 25 }
+    cursor: { type: "ID" }
+    orderBy: { type: "StatusTemplateOrdering", defaultValue: name }
+    orderMode: { type: "OrderingMode", defaultValue: asc }
+  )
+  @refetchable(queryName: "StatusTemplatesLinesRefetchQuery") {
+    statusTemplates(
+      search: $search
+      first: $count
+      after: $cursor
+      orderBy: $orderBy
+      orderMode: $orderMode
+    ) @connection(key: "Pagination_statusTemplates") {
+      edges {
+        node {
+          ...StatusTemplatesLine_node
+        }
+      }
+      pageInfo {
+        endCursor
+        hasNextPage
+        globalCount
+      }
+    }
+  }
+`;
+
+export const StatusTemplatesLineFragment = graphql`
+  fragment StatusTemplatesLine_node on StatusTemplate {
+    id
+    name
+    color
+    usages
+  }
+`;
 
 const LOCAL_STORAGE_KEY = 'status-templates';
 
 const StatusTemplates = () => {
   const { t_i18n } = useFormatter();
   const { setTitle } = useConnectedDocumentModifier();
   setTitle(t_i18n('Status Templates | Taxonomies | Settings'));
-  const classes = useStyles();
-  const { viewStorage, helpers, paginationOptions } = usePaginationLocalStorage<StatusTemplatesLinesPaginationQuery$variables>(
+
+  const initialValues = {
+    searchTerm: '',
+    sortBy: 'name',
+    orderAsc: false,
```

Comment:
should be true, else the status are not sorted by alphabetical order by default: 
![image](https://github.com/user-attachments/assets/23f2bcbd-4930-4f66-9838-a3261e1154c2)